spec/helpers/application_helper/buttons/view_spec.rb

```diff
@@ -31,11 +31,11 @@
     before { subject.calculate_properties }
 
     context 'when gtl_type is set' do
-      it_behaves_like 'a disabled button'
+      include_examples 'ApplicationHelper::Button::Basic#disabled?', nil
```

Comment:
According to [RSpec's documentation](http://www.rubydoc.info/github/rspec/rspec-expectations/RSpec%2FMatchers%3Abe_falsey), `be_falsey` should match `nil`.